scripts/hadolint.bash

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# dockerfile linter tool
+# - https://github.com/hadolint/hadolint
+exec docker run --rm -i hadolint/hadolint < "$@"
```

Comment:
Very nice!
BTW: in scripts we agreed to use  long version of the options e.g. ``docker run --rm --interactive --tty ... ``

Reply:
yeah I copy pasted from shellcheck... will change it...